Scoop/Controller/Onboarding/AboutYouViewController.swift

```diff
@@ -47,78 +47,113 @@ class AboutYouViewController: OnboardingViewController {
             self.delegate?.didTapNext(navCtrl, nextViewController: nil)
         }
         
+        setupTitleLines()
         setupStackView()
+        setBackButtonVisibility(isHidden: true)
         setupNextButton(action: nextAction ?? UIAction(handler: { _ in
             return
         }))
+        
     }
     
     private func setupStackView() {
+        let stackviewMultiplier = 0.10
+        let leadtrailInset = 20.0
+        let spacing = -12
+        let screenSize = UIScreen.main.bounds
+        let labelFont = UIFont(name: "Rambla-Regular", size: 16)
+        let textFieldBorderWidth = 1.0
+        let textFieldCornerRadius = 8.0
+        let textFieldBorderColor = UIColor(red: 0.584, green: 0.616, blue: 0.647, alpha: 1).cgColor
+        let textFieldFont = UIFont(name: "SFPro", size: 16)
+        
         stackView.axis = .vertical
         stackView.distribution = .fill
-        stackView.spacing = 20
         stackView.alignment = .leading
+        stackView.spacing = 24
         view.addSubview(stackView)
         
         stackView.snp.makeConstraints { make in
-            make.leading.trailing.equalToSuperview().inset(60)
-            make.center.equalToSuperview()
+            make.leading.trailing.equalToSuperview().inset(leadtrailInset)
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(stackviewMultiplier * screenSize.height)
         }
         
         let nameLabel = UILabel()
-        nameLabel.font = .systemFont(ofSize: 22)
-        nameLabel.text = "Name"
+        nameLabel.font = .systemFont(ofSize: 16)
+        nameLabel.text = "NAME"
         nameLabel.textColor = .black
         stackView.addArrangedSubview(nameLabel)
         
-        nameTextField.font = .systemFont(ofSize: 22)
         nameTextField.textColor = .darkGray
-        nameTextField.placeholder = "name..."
+        nameTextField.placeholder = "Enter name..."
         stackView.addArrangedSubview(nameTextField)
+        nameTextField.snp.makeConstraints { make in
```

Comment:
I don't think you are using a StackView correctly. The idea behind a StackView is that you shouldn't really be setting the position of a subview relative to another subview in that StackView. The StackView itself should be doing all the positioning. The only exception I can think of this is when you are setting size constraints inside a StackView.